bot/handlers/administration/create_key_handler.py

```diff
@@ -20,16 +22,49 @@ def get_commands(self) -> List[str]:
     def _get_validator_functions(self) -> ValidatorFunctions:
         return [
             self.__check_argument_count,
+            self.__check_days_is_digit,
+            self.__check_key_is_unique,
         ]
 
     async def __check_argument_count(self, message: Message) -> bool:
         return await self._validate_argument_count(
             message, 3, get_create_key_usage_message(),
         )
+
+    async def __check_days_is_digit(self, message: Message) -> bool:
+        args = message.text.split()
+        if not args[1].isdigit():
+            await self.__replay_wrong_argument(message)
+            return False
+        return True
+
+    async def __check_key_is_unique(self, message: Message) -> bool:
+        args = message.text.split()
+        key = " ".join(args[2:])
+        key_exists = await DatabaseManager.get_subscription_days_by_key(key)
+        if key_exists is not None:
+            await self.__replay_key_already_exists(message, key)
+            return False
+        return True
+
     async def _do_handle(self, message: Message) -> None:
         args = message.text.split()
         days = int(args[1])
-        name = " ".join(args[2:])
+        key = " ".join(args[2:])
+
+        await DatabaseManager.create_subscription_key(days, key)
+        await self.__replay_key_added(message, days, key)
+
+    async def __replay_key_added(self, message: Message, days: int, key: str) -> None:
+        await self._answer(message, get_create_key_success_message(days, key))
+        await self._log_system_message(
+            logging.INFO, get_create_key_success_message(days, key),
+        )
+
+    async def __replay_wrong_argument(self, message: Message) -> None:
+        await self._answer(message, get_create_key_usage_message())
+        await self._log_system_message(logging.INFO, get_create_key_usage_message())
 
-        await DatabaseManager.create_subscription_key(days, name)
-        await self._answer(message,get_create_key_success_message(days, name))
+    async def __replay_key_already_exists(self, message: Message, key: str) -> None:
+        await self._answer(message, get_key_already_exists_message(key))
+        await self._log_system_message(logging.INFO, get_key_already_exists_message(key))
```

Comment:
jakie replay XD

Reply:
A z jakiegoś innego pliku zajebałem ten schemat nazwy xD

Reply:
A to jak to nazwać ?

Reply:
reply a nie replay XD

Reply:
Aaaaaa faktycznie kurła nie zauważyłem XD to już widać który kod pisałem ja a który gpt bo gpt takich błędów nie robi XDDD

Reply:
I dlatego drugie oko potrzebne